docs/tutorials/text_prediction/automm.md

```diff
@@ -153,241 +95,211 @@ train_data = train_data.sample(500, random_state=0)
-
-`data` config defines some model-agnostic rules in preprocessing data. Note that AutoMM converts categorical data into text by default.
-
-`optimization` config has hyper-parameters for model training. AutoMM uses layer-wise learning rate decay, which decreases the learning rate gradually from the output to the input end of one model.
+In this example, `AutoMMPredictor` finetunes CLIP with the image, text, and categorical (converted to text) data.
 
-`env` contains the environment/machine related configurations. For example, the optimal values of `per_gpu_batch_size` and `per_gpu_batch_size_evaluation` are closely related to the GPU memory size.
-
-You can flexibly customize any hyper-parameter in `config` via the `hyperparameters` argument of `.fit()`. To access one hyper-parameter in `config`, you need to traverse from top-level keys to bottom-level keys and join them together with `.` For example, if you want to change the per gpu batch size to 16, you can set `hyperparameters={"env.per_gpu_batch_size": 16}`.
-
-
-Next, we will introduce how to customize the default config to train different models.
-
-## Image Prediction
+### TIMM + Huggingface Transformers + More
+In addtion to CLIP, `AutoMMPredictor` can simultaneously finetune various [timm](https://github.com/rwightman/pytorch-image-models) backbones and [huggingface transformers](https://github.com/huggingface/transformers). Moreover, `AutoMMPredictor` uses MLP for numerical data but converts categorical data to text by default. 
 
-Let's first train an image classifier on the PetFinder dataset. AutoMM supports loading diverse [timm](https://github.com/rwightman/pytorch-image-models) backbones. We can choose one image backbone, e.g., [swin_tiny_patch4_window7_224](https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/swin_transformer.py#L69), by using hyper-parameter `model.timm_image.checkpoint_name`.
+Let's use `AutoMMPredictor` to train a late fusion model including [CLIP](https://huggingface.co/openai/clip-vit-base-patch32), [swin_small_patch4_window7_224](https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/swin_transformer.py#L65), [google/electra-small-discriminator](https://huggingface.co/google/electra-small-discriminator), a numerical MLP, and a fusion MLP. 
 
 
 ```{.python .input}
 from autogluon.text.automm import AutoMMPredictor
 predictor = AutoMMPredictor(label=label_col)
 predictor.fit(
     train_data=train_data,
-    config=config,
+    config={
```

Comment:
Would it make sense to have a default value for `config` ? 

```
    config={
        "model": "fusion_mlp_image_text_tabular",
        "data": "default",
        "optimization": "adamw",
        "environment": "default",
    },
```

Reply:
Added.